app/assets/stylesheets/common/admin/customize.scss

```diff
@@ -23,6 +23,22 @@
   }
 }
 
+.settings-editor {
+  .ace-wrapper {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    min-height: 200px;
```

Comment:
This is a little short on desktop. Do we need the absolute positioning here for ACE to be styled properly?

Reply:
made is 300px, looks better.